src/inc/msquic.h

```diff
@@ -661,6 +661,28 @@ typedef struct QUIC_TLS_SECRETS {
     uint8_t ServerTrafficSecret0[QUIC_TLS_SECRETS_MAX_SECRET_LEN];
 } QUIC_TLS_SECRETS;
 
+typedef enum QUIC_FLOW_BLOCK_REASON {
+    QUIC_FLOW_BLOCKED_SCHEDULING            = 0x01,
+    QUIC_FLOW_BLOCKED_PACING                = 0x02,
+    QUIC_FLOW_BLOCKED_AMPLIFICATION_PROT    = 0x04,
+    QUIC_FLOW_BLOCKED_CONGESTION_CONTROL    = 0x08,
+    QUIC_FLOW_BLOCKED_CONN_FLOW_CONTROL     = 0x10,
+    QUIC_FLOW_BLOCKED_STREAM_ID_FLOW_CONTROL= 0x20,
+    QUIC_FLOW_BLOCKED_STREAM_FLOW_CONTROL   = 0x40,
+    QUIC_FLOW_BLOCKED_APP                   = 0x80
+} QUIC_FLOW_BLOCK_REASON;
```

Comment:
Why does this need to move to msquic.h?

Reply:
It doesn't have to be moved here now because we return `QUIC_STREAM_STATISTICS`. Though, if we move this back, we'd have to add duplicated enums in src/tools/etw/trace.c. Worth it?

Reply:
I'd rather not expose this publicly.